### PR TITLE
JAVADOCS UPDATED

### PR DESCRIPTION
**Pull request by aditi-v79**

### CODE REVIEW

**Comment by Surya-06** [comment]
Please do not merge without merging with latest main. This is required so that all the docs are updated.

File: app/src/main/java/com/xa/xpensauditor/ShowTransActivity.java [hunk 1]
```diff
@@ -18,6 +18,9 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
```
Comment (kaushikjadhav01) [inline app/src/main/java/com/xa/xpensauditor/ShowTransActivity.java:21]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.

File: app/src/main/java/com/xa/xpensauditor/AddTransactionActivity.java [hunk 2]
```diff
@@ -271,7 +271,7 @@
 
 
                 RefTran1.addChildEventListener(new com.firebase.client.ChildEventListener() {
-                    String amount, cat, shname, shDay, shMonth, shYear;
+                    String amount, cat, shname, shDay, shMonth, shYear,x ;
```
Comment (kaushikjadhav01) [inline app/src/main/java/com/xa/xpensauditor/AddTransactionActivity.java:274]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.coding.MultipleVariableDeclarationsCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Each variable declaration must be in its own statement.

Comment (kaushikjadhav01) [inline app/src/main/java/com/xa/xpensauditor/AddTransactionActivity.java:274]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>',' is not followed by whitespace.

Comment (kaushikjadhav01) [inline app/src/main/java/com/xa/xpensauditor/AddTransactionActivity.java:274]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.MemberNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Member name 'x' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'.

Comment (kaushikjadhav01) [inline app/src/main/java/com/xa/xpensauditor/AddTransactionActivity.java:274]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.NoWhitespaceBeforeCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>';' is preceded with whitespace.

File: app/src/main/java/com/xa/xpensauditor/NavShowAnalysis.java [hunk 3]
```diff
@@ -54,6 +54,9 @@
 import com.google.firebase.database.FirebaseDatabase;
 import com.google.firebase.database.ValueEventListener;
 
+/**
```
Comment (kaushikjadhav01) [inline app/src/main/java/com/xa/xpensauditor/NavShowAnalysis.java:57]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.

File: app/src/main/java/com/xa/xpensauditor/SearchProduct.java [hunk 4]
```diff
@@ -29,6 +29,9 @@
 import java.util.Arrays;
 import android.app.ProgressDialog;
 
+/**
```
Comment (kaushikjadhav01) [inline app/src/main/java/com/xa/xpensauditor/SearchProduct.java:32]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.

File: app/src/main/java/com/xa/xpensauditor/ResetPasswordActivity.java [hunk 5]
```diff
@@ -16,6 +16,9 @@
 import com.google.android.gms.tasks.Task;
 import com.google.firebase.auth.FirebaseAuth;
 
+/**
```
Comment (kaushikjadhav01) [inline app/src/main/java/com/xa/xpensauditor/ResetPasswordActivity.java:19]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.

File: app/src/main/java/com/xa/xpensauditor/SetLimitActivity.java [hunk 6]
```diff
@@ -22,6 +22,9 @@
 
 import java.util.Calendar;
 
+/**
```
Comment (kaushikjadhav01) [inline app/src/main/java/com/xa/xpensauditor/SetLimitActivity.java:25]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.

File: app/src/main/java/com/xa/xpensauditor/ProfileActivity.java [hunk 7]
```diff
@@ -30,7 +30,9 @@
 
 public class ProfileActivity extends AppCompatActivity {
 
-
+    /**
```
Comment (kaushikjadhav01) [inline app/src/main/java/com/xa/xpensauditor/ProfileActivity.java:33]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.

**Pull request merged**